### PR TITLE
Fix styles for KtHeading to make them scoped

### PR DESCRIPTION
**Pull request by santiagoballadares**

### CODE REVIEW

File: packages/documentation/pages/foundations/color.vue [hunk 1]
```diff
@@ -149,3 +149,22 @@ export default {
 	},
 }
 </script>
+
+<style lang="scss" scoped>
```
Comment (FlorianWendelborn) [inline packages/documentation/pages/foundations/color.vue:153]: Why is this copy-pasted here?

File: packages/documentation/pages/foundations/text/typography.vue [hunk 2]
```diff
@@ -149,3 +149,22 @@ export default defineComponent({
 	name: 'DocumentationPageFoundationsTextTypography',
 })
 </script>
+
+<style lang="scss" scoped>
```
Comment (FlorianWendelborn) [inline packages/documentation/pages/foundations/text/typography.vue:153]: Why is this copy-pasted here?

**Review by FlorianWendelborn — CHANGES_REQUESTED** [review]

File: packages/documentation/pages/foundations/color.vue (hunk 1 above)
Comment (santiagoballadares) [reply packages/documentation/pages/foundations/color.vue:153]: This component is also using the styles that are now scoped to KtHeading.

File: packages/documentation/pages/foundations/text/typography.vue (hunk 2 above)
Comment (santiagoballadares) [reply packages/documentation/pages/foundations/text/typography.vue:153]: This component is also using the styles that are now scoped to KtHeading.

File: packages/documentation/pages/foundations/color.vue (hunk 1 above)
Comment (FlorianWendelborn) [reply packages/documentation/pages/foundations/color.vue:153]: Why does this component not use KtHeading then?

File: packages/documentation/pages/foundations/text/typography.vue (hunk 2 above)
Comment (FlorianWendelborn) [reply packages/documentation/pages/foundations/text/typography.vue:153]: Why does this component not use KtHeading then?

**Comment by FlorianWendelborn** [comment]
Just noticed your git email seems to be misconfigured. There's a section in the readme on how to set it up properly

**Comment by FlorianWendelborn** [comment]
@santiagoballadares I rebased, squashed, fixed the commit message, and commit author. Will most likely merge now.

Please, execute `git config user.email "744091+santiagoballadares@users.noreply.github.com"` in the kotti repository for future commits, thanks.

**Review by FlorianWendelborn — APPROVED** [review]

**Pull request merged**